aws/cloudformation/metaflow-cfn-template.yml

```diff
@@ -90,6 +95,7 @@ Mappings:
 Conditions:
   EnableAuth: !Equals [ !Ref APIBasicAuth, 'true']
   EnableRole: !Equals [ !Ref CustomRole, 'true']
+  BatchFargate: !Equals [ !Ref BatchType, 'fargate']
```

Comment:
minor nit: we can rename it to `EnableFargateOnBatch` for consistency.

Reply:
Updated.